### ISSUE TITLE
Allow outline but deny ouline-offset for IE 11

### ISSUE THREAD

**Issue by cronfy** [anandthakker/doiuse#70]
> Unexpected browser feature "outline" is only partially supported by IE 11 (plugin/no-unsupported-browser-features

According to caniuse: http://caniuse.com/#feat=outline

`outline` is supported in IE 11, only `outline-offset` is not supported.

Is there any way to allow `outline` and deny only `outline-offset`?

**Comment by ismay** [comment]
See here: https://github.com/anandthakker/doiuse#ignoring-file-specific-rules

Does ignoring `outline` do you what you want? I.e.: `/* doiuse-disable outline */`. Also, I have to remind you that issues are not for support requests but for reporting bugs and feature requests. If you have a support request then please use stack overflow.

If you're reporting a bug, please open either a pull request with a failing test demonstrating the
problem, or include a link to a repository with a reduced test case demonstrating the problem.

**Comment by cronfy** [comment]
> Does ignoring outline do you what you want?

No, because it will ignore `outline-offset` then, but I want to warn about it.

> issues are not for support requests but for reporting bugs and feature requests

Well, I intended it to be a feature request/bug report, becase current behaviour either produces false positives or does not report important issues:

1. If `outline` is not disabled, and there is `outline: none`, doiuse warns about `Unexpected browser feature "outline"`, although `outline: none` is completely supported.
2. If `outline` is disabled, and there is `outline-offset`, doiuse produces no warning, although `outline-offset` is not supported in IE11.

Same about `appearance` #71.

**Comment by ismay** [comment]
> Well, I intended it to be a feature request/bug report, because current behaviour either produces false positives or does not report important issues

Ok, my apologies I misinterpreted your initial report. So indeed seems related to #71
